setup-env/action.yaml

```diff
@@ -44,5 +44,5 @@ runs:
         touch .env;
       shell: bash
     - name: Create more disk space # See https://github.com/actions/runner-images/issues/2840#issuecomment-790492173
-      run: sudo rm -rf /usr/share/dotnet && sudo rm -rf /opt/ghc && sudo rm -rf "/usr/local/share/boost" && sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+      run: rm -rf /usr/share/dotnet && rm -rf /opt/ghc && rm -rf "/usr/local/share/boost" && rm -rf "$AGENT_TOOLSDIRECTORY"
```

Comment:
@ShanaLMoore since this touches not just the deploy workflow but all of them, I would like to know if this update was also tested against the other actions to ensure its still working. It's odd to me that they run on the same runner as the other actions, why is it erroring on sudo su for the github ci runner

Reply:
oops sorry @aprilrieger I tagged you in the wrong PR. I meant to close this one and ask for your review here, which I simply just reverted the deploy changes: 

https://github.com/scientist-softserv/palni-palci/pull/1018

Reply:
@aprilrieger  But initially I tried removing sudo from all the actions and referencing this branch from pals, but it still wasn't working. It seemed like pals didn't actually get the update or something because the build still failed with the sudo error.

Reply:
Got it! Thanks I see the other one now :)

Reply:
I have a feeling we may hit this again on other projects soon. If so, we should keep this PR in mind as it may be the path forward

Reply:
Its due to the deploy action container being a helm container -- which is why its specific to that action only.

Reply:
https://github.com/scientist-softserv/actions/pull/72/files

Reply:
Nice!